.github/workflows/deploy.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - uses: nttld/setup-ndk@v1
         with:
-          ndk-version: r23c
+          ndk-version: r22b
```

Comment:
downgrades like this really should be mentioned, and _explained_ in the PR description. This is not what someone would expect to happen in this PR coming from the title and description!